src/en/callbacks.md

```diff
@@ -34,7 +34,7 @@ This functionality works with test API keys, but does not work with smoke testin
 
 The callback message is formatted in JSON. All of the values are strings. The key, description and format of the callback message arguments will be:
 
-|Key | Description | Format|
+|Key string | Description | Format|
```

Comment:
I believe for JSON, the Key value is always a string. In the same vein however, we could update the header to say `String format` to indicate that values associated with a key are strings 🙂

Reply:
Oo thanks! will do!